style.css

```diff
@@ -686,3 +688,136 @@ hr {
   color: #dfe1e6;
   margin-left: 20px;
 }
+
+.card2 {
+  width: 375px;
+  height: 779px;
+  background-color: #fff;
+  margin: auto;
+}
+
+.content4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('./contact-images/contact form shapes-mobile-bg.svg');
+  background-repeat: no-repeat;
+  background-attachment: absolute;
+  background-position: right;
+  margin-top: 0;
+  width: 375px;
+  height: 779px;
+  border-radius: 80px 0 0 0;
+  background-color: #6070ff;
+}
+
+.contact-header {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding-top: 120px;
+  color: #fff;
+  font-family: "poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.p-contact {
+  width: 327px;
+  height: 102px;
+  font-family: "poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: 10px;
+  padding-bottom: 65px;
+  color: #ebebff;
+}
+
+.form-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  margin-bottom: 90px;
+  align-items: center;
+}
+
+#name {
+  height: 50px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+#mail {
+  height: 50px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
```

Comment:
- [ ] As the previous code reviewer mentioned, kindly adjust the width of your inputs so they can have a consistent size and your form can look more professional like the Figma template. I suggest you use the same width value you used for the `textarea`.
 - [OPTIONAL] It would be nice if you remove the whitespace below the form section so there are no unused spaces and your page looks cleaner.
 -Your design: 
<img width="486" alt="image" src="https://user-images.githubusercontent.com/61919617/194177534-0c0056c4-6faa-49e4-b0bd-0f2deb8eb631.png">